conf/messages

```diff
@@ -1064,8 +1064,8 @@ disposalDate.link=<a href="{0}" target="_blank">Find out what happens on the exc
 disposalDate.error.required=Enter a contract exchange date
 disposalDate.error.invalid=Enter a real contract exchange date
 disposalDate.error.tooFarInFuture=The contract exchange date must be today or in the past
-disposalDate.error.periodOfAdminDeathNotAfterDate=The contract exchange date cannot be on or before the date of death. Go ‘back’ and check your answer to the question ‘Who are you completing this return for?’
-disposalDate.error.nonPeriodOfAdminDeathAfterDate=The contract exchange date cannot be after the date of death. Go ‘back’ and check your answer to the question ‘Who are you completing this return for?’
+disposalDate.error.periodOfAdminDeathNotAfterDate=The exchange date must be on or after the date of death
+disposalDate.error.nonPeriodOfAdminDeathAfterDate=The exchange date must be on or before the date of death
```

Comment:
Are they approved from Tom? Because they are existing messages, not new.

Reply:
Yes, Jack has been contacting Sujeeth and Tom and these are already finalized ones.

Reply:
If they are okay, am okay because cannot test it functionally.